lib/cloud/test/aws.go

```diff
@@ -14,11 +14,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package cloud
```

Comment:
I think this package should be renamed to `mocks` 
It scopes the logic better than `test`

Reply:
Ya, we had a small discussion in the previous PR and it does appear it's more a `mocks`. Updated now.